Library/Homebrew/cmd/untap.rb

```diff
@@ -1,29 +1,31 @@
-require 'cmd/tap' # for tap_args
+require 'cmd/tap' # for tap_args and show_taps
 
 module Homebrew
   def untap
-    raise "Usage is `brew untap <tap-name>`" if ARGV.empty?
+    if ARGV.empty?
+      show_taps
```

Comment:
Sorry, but in what way? `brew tap` begins with the same check of empty arguments.

My default rule of thumb is this: if a command-line tool can do something useful with a no-argument call, do that instead of just throwing an error. That said, I can remove this if people feel strongly against it.